packages/docs/pages/search.tsx

```diff
@@ -15,64 +15,70 @@ const data = [
-                  });
-                };
+      <Panel header="Implementation" headerId="implementation">
+        <CodePreview scope={{ data }}>
+          {/* jsx-to-string:start */}
+          {function Example() {
+            const [items, setItems] = useState(data);
+            const [searchValue, setSearchValue] = useState('');
+            const onChange = (event: React.ChangeEvent<HTMLInputElement>) => setSearchValue(event.target.value);
 
-                return (
-                  <>
-                    <Box marginBottom="medium">
-                      <Search value={searchValue} onChange={onChange} onSubmit={onSubmit} />
-                    </Box>
-                    <Table
-                      columns={[
-                        { header: 'Sku', hash: 'sku', render: ({ sku }) => sku },
-                        { header: 'Name', hash: 'name', render: ({ name }) => name },
-                        { header: 'Stock', hash: 'stock', render: ({ stock }) => stock },
-                      ]}
-                      items={items}
-                    />
-                  </>
-                );
-              }}
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-        </>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => <SearchPropTable />,
-    },
-  ];
+            const onSubmit = () => {
+              setItems((prevItems) => {
+                if (searchValue) {
+                  return prevItems.filter((item) => item.name.includes(searchValue));
+                }
 
-  return (
-    <>
-      <H1>Search</H1>
+                return data;
+              });
+            };
+
+            return (
+              <>
+                <Box marginBottom="medium">
+                  <Search value={searchValue} onChange={onChange} onSubmit={onSubmit} />
+                </Box>
+                <Table
+                  columns={[
+                    { header: 'Sku', hash: 'sku', render: ({ sku }) => sku },
+                    { header: 'Name', hash: 'name', render: ({ name }) => name },
+                    { header: 'Stock', hash: 'stock', render: ({ stock }) => stock },
+                  ]}
+                  items={items}
+                />
+              </>
+            );
+          }}
+          {/* jsx-to-string:end */}
+        </CodePreview>
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <SearchPropTable renderPanel={false} />
+      </Panel>
 
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={[
+            <>Make the search bar easily noticable.</>,
+            <>Always use a search icon within the input box to indicate search functionality.</>,
+          ]}
+          discouraged={[<>Avoid using a search bar when there is small, easily navigable amount of data on a page.</>]}
```

Comment:
Let's just use string here
```suggestion
          recommended={[
            'Make the search bar easily noticable.',
            'Always use a search icon within the input box to indicate search functionality.',
          ]}
          discouraged={['Avoid using a search bar when there is small, easily navigable amount of data on a page.']}
```